assets/js/components/indicators.js

```diff
@@ -201,12 +201,15 @@ export class IndicatorSection {
       const response = await $.ajax({
         url: '/api/geography/geography/'
       });
-      const miifGrouped  = _.groupBy(response.results, "miif_category");
-      let similarGroup = miifGrouped[this.geography.miif_category];
 
-      // Remove current muni from selection
+      var filteredResults = response.results.filter(function( obj ) {
+          return obj.is_disestablished !== true;
+      });
+      const miifGrouped  = _.groupBy(filteredResults, "miif_category");
+      let similarGroup = miifGrouped[this.geography.miif_category];
+      // Remove current muni from selection and disestablished municipalities
```

Comment:
looks like this comment is now out of date - I think you moved the disestablished filter upwards